lib/Transforms/IPO/StructFieldCacheAnalysisImpl.h

```diff
@@ -70,6 +71,17 @@ class StructHotnessAnalyzer {
   void addStruct(const StructFieldAccessInfo *SI);
   void generateHistogram();
   bool isHot(const StructFieldAccessInfo *SI) const;
+  ProfileCountType getMaxHotness() const { return MaxHotness; }
+  Optional<ProfileCountType> getHotness(const StructType *ST) const {
+    Optional<ProfileCountType> Ret;
+    auto it = StructHotness.find(ST);
+    if (it == StructHotness.end())
+      return Ret;
```

Comment:
I didn't know the keyword None of Optional type. That could simply a lot of the code. I'll fix it. Thanks!